modules.json

```diff
@@ -167,39 +167,6 @@
                     }
                 }
             },
-            "subworkflows": {
-                "nf-core": {
-                    "bam_sort_stats_samtools": {
-                        "branch": "master",
-                        "git_sha": "a64788f5ad388f1d2ac5bd5f1f3f8fc81476148c",
-                        "installed_by": ["subworkflows"]
-                    },
-                    "bam_stats_samtools": {
-                        "branch": "master",
-                        "git_sha": "a64788f5ad388f1d2ac5bd5f1f3f8fc81476148c",
-                        "installed_by": ["bam_sort_stats_samtools"]
-                    }
-                }
```

Comment:
these modules have been removed in the modules json, probably by accident. easiest is probably to just reinstall them